Source/EasyNetQ/Extentions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace EasyNetQ
+{
+    public static class Extentions
+    {
+        public static void SafeDispose(this IDisposable disposable)
```

Comment:
I think we can ask directly to the rabbitmq.client team why this? actually dispose should never throw exception:

[CA1065: Do not raise exceptions in unexpected locations](https://msdn.microsoft.com/en-us/library/bb386039.aspx)
> A IDisposable.Dispose method should not throw an exception. Dispose is often called as part of the clean up logic in a finally clause. Therefore, explicitly throwing an exception from Dispose forces the user to add exception handling inside the finally clause.
The Dispose(false) code path should never throw exceptions, because this is almost always called from a finalizer.